chapter2/tests/t20/terraform/dev/route53.tf

```diff
@@ -0,0 +1,30 @@
+resource "aws_route53_record" "records" {
+  zone_id = data.aws_route53_zone.domain.zone_id
+  name    = var.site_subdomain_part
+  type    = "A"
+
+  alias {
+    name                   = module.alb.lb_dns_name
+    zone_id                = module.alb.lb_zone_id
+    evaluate_target_health = true
+  }
+  depends_on = [module.alb]
+}
+
+
+resource "aws_route53_record" "tf-maxim-omelchenko" {
```

Comment:
don't name resource instances after concret project specific names - they will complexify the maintenance in the future

Reply:
> don't name resource instances after concret project specific names - they will complexify the maintenance in the future

@dmitry-mightydevops
Done